docs/pages/build-reference/infrastructure.mdx

```diff
@@ -11,7 +11,9 @@ import { BuildResourceList } from '~/ui/components/utils/infrastructure';
 
 ## Builder IP addresses
 
-Here is the [up-to-date list of builder IP addresses](https://expo.dev/eas-build-worker-ips.txt).
+A list of the IP addresses of the build servers is available [here](https://expo.dev/eas-build-worker-ips.txt). We do not expect to change the list often. Every time we improve the infrastructure and do need to update the IPs, we will also update the timestamp in the header of the file.
```

Comment:
```suggestion
A list of the IP addresses of the build servers is available [here](https://expo.dev/eas-build-worker-ips.txt). We do not expect to change the list often. Every time we improve the infrastructure or update the IPs, we also update the **Last updated** date displayed in this document's footer.
```

A couple of suggestions to clearly point out where the Last updated date is shown for this doc. We don't expect people to check out the frontmatter of the file by going through the `.mdx` file of this doc (We also have **Last updated** date stamp displayed for most of the docs -- something we added recently).

Reply:
I'm not sure I understand? The "file header" in question is this
<img width="289" alt="Zrzut ekranu 2024-06-25 o 18 54 55" src="https://github.com/expo/expo/assets/1151041/c6e34c80-e079-4cfc-8291-9e0728eef48c">

Reply:
My bad, I assumed you meant the updating the date for docs page and didn't know we had a date stamp in the IP address file 🤦. Thanks for the clarification!